### PR TITLE
You can now place material iron tiles in space

### PR DESCRIPTION
**Pull request by Aliceee2ch**
## About The Pull Request

After #87587 i considered that players (especially newbies) can confuse iron tile and material iron tile.

## Why It's Good For The Game

This PR removes barely noticeable difference between iron tile and material iron tile, since they use the same amount of the same material and should therefore act the same.
image

## Changelog

:cl:
fix: you can now place material iron tiles in space like regular iron tiles
/:cl:

### CODE REVIEW

**Review by Fikou — CHANGES_REQUESTED** [review]
we are not doing name checks. look if the iron material is actually there.

File: code/game/turfs/open/space/space.dm [hunk 1]
```diff
@@ -144,7 +144,7 @@ GLOBAL_LIST_EMPTY(starlight)
 		return
 	if(istype(C, /obj/item/stack/rods))
 		build_with_rods(C, user)
-	else if(istype(C, /obj/item/stack/tile/iron))
+	else if(istype(C, /obj/item/stack/tile/iron) || istype(C, /obj/item/stack/tile/material) && C.has_material_type(/datum/material/iron))
```
Comment (SyncIt21) [inline code/game/turfs/open/space/space.dm:147]:
```suggestion
	else if(C.has_material_type(/datum/material/iron))
```

Both tile types have iron.

**Review by SyncIt21 — CHANGES_REQUESTED** [review]

File: code/game/turfs/open/space/space.dm [hunk 2]
```diff
@@ -144,7 +144,7 @@ GLOBAL_LIST_EMPTY(starlight)
 		return
 	if(istype(C, /obj/item/stack/rods))
 		build_with_rods(C, user)
-	else if(istype(C, /obj/item/stack/tile/iron))
+	else if(C.has_material_type(/datum/material/iron))
```
Comment (SyncIt21) [inline code/game/turfs/open/space/space.dm:147]:
```suggestion
	else if(istype(C, /obj/item/stack/tile) && C.has_material_type(/datum/material/iron))
```

My bad. We don't want any item made of iron but only tiles made of iron to be used.

**Review by SyncIt21 — CHANGES_REQUESTED** [review]
I made a mistake in my last review.

**Review by Fikou — CHANGES_REQUESTED** [review]
revert syncits suggestions, they allow stuff like catwalks or reinforced glass tiles to work.

**Pull request merged**